src/draw/sw/blend/lv_draw_sw_blend.c

```diff
@@ -59,7 +59,7 @@ void lv_draw_sw_blend(lv_draw_unit_t * draw_unit, const lv_draw_sw_blend_dsc_t *
 
     LV_PROFILER_BEGIN;
     lv_layer_t * layer = draw_unit->target_layer;
-    uint32_t layer_stride_byte = lv_draw_buf_width_to_stride(lv_area_get_width(&layer->buf_area), layer->color_format);
```

Comment:
@kisvegabor Is `lv_draw_buf_width_to_stride()` useless in all the cases?
I remember for layers, the stride had to be calculated here, hence this code.

Reply:
I recommend having a test with layer (like 100x100 and a stride 128), test should pass with SW render as well with GPU render.

Reply:
I've run the lv_demo_benchmark to check for regressions. I'll revisit this later; I have to get a better understanding of the layers part.